src/main/java/frc/robot/Constants.java

```diff
@@ -35,44 +35,44 @@ public static final class DriveConstants {
 
         public static final int frontLeftDriveID = 1;
 		public static final int backLeftDriveID = 3;
-		public static final int frontRightDriveID = 7;
 		public static final int backRightDriveID = 5;
+		public static final int frontRightDriveID = 7;
 
-        public static final int frontLeftRotationMotorID = 2; 
-        public static final int frontRightRotationMotorID = 4; 
-        public static final int backLeftRotationMotorID = 8; 
-        public static final int backRightRotationMotorID = 6; 
+        public static final int frontLeftRotationMotorID = 2;
+        public static final int frontRightRotationMotorID = 4;
+        public static final int backRightRotationMotorID = 6;
+        public static final int backLeftRotationMotorID = 8;
 
         public static final int frontLeftRotationEncoderID = 1; 
         public static final int frontRightRotationEncoderID = 2; 
+        public static final int backRightRotationEncoderID = 3;
         public static final int backLeftRotationEncoderID = 4; 
-        public static final int backRightRotationEncoderID = 3; 
 
         public static final double frontLeftAngleOffset = 0.739378739760879;
         public static final double frontRightAngleOffset = 2.555611992617478; 
-        public static final double backLeftAngleOffset = 2.551010050253821;
-        public static final double backRightAngleOffset = 5.533068701903508;
+        public static final double backRightAngleOffset = 5.533068701903508; // TODO: Might have to switch these
+        public static final double backLeftAngleOffset = 2.551010050253821; // TODO: Might have to switch these
 
 		public static final double autoDrivePercent = 0.6;
         public static final double driveWheelGearReduction = 6.12; // placeholder
         public static final double wheelRadiusM = Units.inchesToMeters(2);
 
         public static final SwerveDriveKinematics kinematics = 
             new SwerveDriveKinematics(
-                new Translation2d(trackWidth / 2.0, wheelBase / 2.0), //front left placeholder
-                new Translation2d(trackWidth / 2.0, -wheelBase / 2.0), //front right placeholder
-                new Translation2d(-trackWidth / 2.0, wheelBase / 2.0), //rear left placeholder
-                new Translation2d(-trackWidth / 2.0, -wheelBase / 2.0) //rear right placeholder
+                new Translation2d(-trackWidth / 2.0, wheelBase / 2.0), //TODO: Test
+                new Translation2d(trackWidth / 2.0, wheelBase / 2.0), //TODO: Test
+                new Translation2d(trackWidth / 2.0, -wheelBase / 2.0), //TODO: Test
+                new Translation2d(-trackWidth / 2.0, -wheelBase / 2.0) //TODO: Test
         );
```

Comment:
I think this needs to be FL, FR, BL, BR per: https://docs.wpilib.org/en/stable/docs/software/kinematics-and-odometry/swerve-drive-kinematics.html

Reply:
Sorry, my bad